### ISSUE TITLE
TextEditor.scopesForBufferPosition is deprecated.

### ISSUE THREAD

**Issue by Kev** [yasuyuky/autocomplete-clang#36]
(Generated automatically by Deprecation Cop)

Use ::scopeDescriptorForBufferPosition instead. The return value has changed! It now returns a `ScopeDescriptor`
```
TextEditor.scopesForBufferPosition (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:2436:7)
Object.getFirstCursorScopes (.../autocomplete-clang/lib/util.coffee:20:22)
```

**Comment by yasuyuky** [comment]
Thanks. I will try to fix this..

**Comment by yasuyuky** [comment]
Fixed in v0.6.5. Thanks
